### PR TITLE
Release 0.14.0

### PR DESCRIPTION
**Pull request by cdrage**
ping @beardypig @gravyboat

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,105 @@
...
+
+- Multiple plugin fixes
+- Bug fixes for DASH streams (https://github.com/streamlink/streamlink/pull/1846)
+- Updated API call for api.utils hours_minutes_seconds (https://github.com/streamlink/streamlink/pull/1804)
+- Updated documentation (https://github.com/streamlink/streamlink/pull/1826)
+- Dict structures fix (https://github.com/streamlink/streamlink/pull/1792)
+- Reformated help menu (https://github.com/streamlink/streamlink/pull/1754)
+- Logger fix (https://github.com/streamlink/streamlink/pull/1773)
+
+## Installation
```
[the remark this answers is not in the document]
Comment (cdrage) [reply CHANGELOG.md]: Crap I keep forgetting haha. I'll remove it

[the remark this answers is not in the document]
Comment (cdrage) [reply CHANGELOG.md]: What about the supporting text? (open collective)

**Comment by cdrage** [comment]
@beardypig Updated, feel free to merge!

**Comment by beardypig** [comment]
Now we just need a tag on 4f5bb1e453ef236d9075d34f4d5de7a77d6fb313 :

**Comment by cdrage** [comment]
@beardypig I really need to write this down, but anything else I need to do other than having this PR merged, or do I create a tag (do I have to paste in the changelog into there too)?

I'll post a PR later adding documentation somewhere for someone else to do this in the future too.

**Comment by beardypig** [comment]
Just push a tag to the repo for that commit and everything will get sorted out :)

**Comment by cdrage** [comment]
@beardypig Tag is pushed, just waiting for the release to be pushed out too (unless it's done manually?)

**Comment by beardypig** [comment]
Look good to me, we have 

- [The release](https://github.com/streamlink/streamlink/releases/tag/0.14.0)
- [PyPI release](https://pypi.org/project/streamlink/)
- [Updated Docs](https://streamlink.github.io/)

**Comment by bastimeyer** [comment]
Hm... :worried: 
https://travis-ci.org/streamlink/streamlink/jobs/396916100#L3358

**Comment by beardypig** [comment]
>Hm... :worried:  ...

@bastimeyer that was a re-run :)

**Comment by gravyboat** [comment]
@beardypig Okay so let's clarify the steps real quick here just so we have them for future reference (should probably modify the readme now that I think about it).

1. User clones the repo, then from master makes their own branch with the release, then runs `https://github.com/streamlink/streamlink/blob/master/script/release.sh`
2. User creates a PR with the updated changelog information
3. Once merged, user creates a tag off of the appropriate commit that was merged to master

Everything after that is handled automatically and the person doing the release doesn't need to do anything else right?

**Comment by cdrage** [comment]
@gravyboat It was a bit odd since I saw @beardypig create the release draft (I thought it was created automatically).

So the extra step is to create the tag as well as publish a release draft (what I did this time, is simply `git tag 0.14.0` and then `git push upstream 0.14.0`) which didn't create a release draft / publish.

**Comment by cdrage** [comment]
@beardypig Ah! That makes sense. So literally just pushing a PR with CHANGELOG updates and tag is all we need. I'll note that down :+1:

**Comment by beardypig** [comment]
> Should we create a separate user called streamlink-bot then on GitHub so it's kinda no-ones-name on there?

@cdrage that's what I did :) Where is that line in the docs?

**Comment by beardypig** [comment]
@cdrage, if you want you can merge #1859 and we can do a `0.14.1` release. There are only a few bug fixes, nothing major. We can also test out the `streamlink-bot` :)

**Pull request merged**